src/components/ble/BatteryInformationService.h

```diff
@@ -20,20 +20,25 @@ namespace Pinetime {
 
       int OnBatteryServiceRequested(uint16_t attributeHandle, ble_gatt_access_ctxt* context);
       void NotifyBatteryLevel(uint16_t connectionHandle, uint8_t level);
+      void NotifyBatteryCharging(uint16_t connectionHandle, bool isCharging);
 
     private:
       Controllers::Battery& batteryController;
       static constexpr uint16_t batteryInformationServiceId {0x180F};
       static constexpr uint16_t batteryLevelId {0x2A19};
+      static constexpr uint16_t isChargingId {0x2A1B};
```

Comment:
I can't find this number assigned in https://www.bluetooth.com/wp-content/uploads/Files/Specification/HTML/Assigned_Numbers/out/en/Assigned_Numbers.pdf

I could totally be missing it (I tried CTRL-F!), could you point me to it

Reply:
That was my fault; I screwed up the number. Should be good now.